services/core-web/src/components/Forms/ExplosivesPermit/ExplosivesPermitForm.tsx

```diff
@@ -400,11 +411,10 @@ const mapDispatchToProps = (dispatch) =>
   );
 
 export default compose(
-  connect(mapStateToProps, mapDispatchToProps),
+  connect<StateProps, any, ExplosivesPermitFormProps>(mapStateToProps, mapDispatchToProps),
   reduxForm({
     form: FORM.EXPLOSIVES_PERMIT,
     touchOnBlur: true,
-    validate: validateBusinessRules,
     onSubmitSuccess: resetForm(FORM.EXPLOSIVES_PERMIT),
   })
-)(ExplosivesPermitForm);
+)(ExplosivesPermitForm) as FC<ExplosivesPermitFormProps>;
```

Comment:
Was this the Typescript error you were getting that was hard to figure out?  Redux is dumb.